specifications/alcb-token.md

```diff
@@ -0,0 +1,117 @@
+---
+title: ALCB Token Specification
+author: chgorman
+status: Draft
+created: 2022-11-23
+---
+
+# ALCB Token Specification
+
+The ALCB Token is the utility token of AliceNet.
+It follows a bonding curve which is parameterized by
+the total ETH reserves and specifies the ALCB/ETH (inverse price);
+this is different from the standard parameterization,
+which tracks the total token supply and specifies
+the ETH/Token (price).
+
+## Minting and Burning
```

Comment:
Probably be a good idea to have the example chart in here too to help follow along with the math.

Reply:
Examples for showing how many tokens are received when purchasing at different ETH reserves?

Reply:
Added some specific numbers. Need to discuss more.